http/reactive-routes/src/main/java/io/quarkus/ts/http/reactiveroutes/validation/ValidationOnResponseRouteHandler.java

```diff
@@ -25,13 +24,6 @@ Uni<Response> validateUniResponseWithTypeReturnsInvalidID() {
         return Uni.createFrom().item(response);
     }
 
-    @Route(methods = HttpMethod.GET, path = "/response-uni-invalid-string")
-    @Valid
```

Comment:
Doc

https://quarkus.io/guides/reactive-routes#using-bean-validation

Could try to.... return a class that contains a string and... add the @Valid and check is working?

Reply:
https://github.com/quarkusio/quarkus/issues/15168#issuecomment-782894044

Reply:
```
@valid
Uni<CustomResponse> validateUniResponseWithStringReturnsInvalidSize() {
        return Uni.createFrom().item(new CustomResponse("ASDASD"));
    }
```

```
CustomResponse {
@Size(min = 3, max = 3, message = "response must have 3 characters")
private String content;

public CustomResponse(String s) {

this.content = s;
} 
}
```

Reply:
Well, this kills the whole idea of the test in the first place(to check working inside uni).

Also a check like this already exists inside neighbour methods(see [1])

[1] https://github.com/quarkus-qe/quarkus-test-suite/blob/main/http/reactive-routes/src/main/java/io/quarkus/ts/http/reactiveroutes/validation/Response.java